jib-core/src/test/java/com/google/cloud/tools/jib/cache/LayerEntriesSelectorTest.java

```diff
@@ -126,4 +150,26 @@ public void testGenerateSelector_fileModified() throws IOException {
     Assert.assertEquals(
         expectedSelector, LayerEntriesSelector.generateSelector(ImmutableList.of(layerEntry)));
   }
+
+  @Test
+  public void testGenerateSelector_permissionsModified() throws IOException {
+    Path layerFile = temporaryFolder.newFolder("testFolder").toPath().resolve("file");
+    Files.write(layerFile, "hello".getBytes(StandardCharsets.UTF_8));
+    LayerEntry layerEntry111 =
+        new LayerEntry(
+            layerFile,
+            AbsoluteUnixPath.get("/extraction/path"),
+            FilePermissions.fromOctalString("111"));
+    LayerEntry layerEntry222 =
+        new LayerEntry(
+            layerFile,
+            AbsoluteUnixPath.get("/extraction/path"),
+            FilePermissions.fromOctalString("222"));
+
+    // Verify that changing permissions generates a different selector
+    DescriptorDigest expectedSelector =
```

Comment:
The `expected` naming makes me to think that the actual should be equal to this. `entry111Selector` or just `selector`?

Reply:
Ah right, I think that was the result of some copy-pasting. Actually I don't think the variable is even really necessary.